### DISCUSSION TITLE
Bluetooth Buzzer connection esp to raspberry

### DISCUSSION THREAD

**Discussion by Striker62** [photobooth-app/photobooth-app#239] [Q&A] [answered]
I managed to flash my AZ-Delivery ESP32-WROOM-32 successfully .
Serial monitor does not show any issues.

Bluetooth is activated on my raspberry pi 4 Bullseye ( I learned that you prefer to use Bookworm ;-) that will be my next step )
When I press add new device on my raspberry pi the esp32 is found when I press pair, i can see in the serial monitor of the esp32
BLE ready
Client has conneted 

when I press the button attached to the esp32 i can see the letter c in the esp32 monitor

But the raspberry pi  show the message "Pairing failed - Timeout was reached.

Any Idea how I can pair the both ?

**Comment by Striker62** [comment]
sorry I did not realized that the Buzzer was already connected to my PC.
Now I switch off bluetooth on my pc.

When I try to connect esp to raspberry I get now following message 
Pairing failed -  GDBuss.Error.org.bluez.Error.ConnectionAttemptFailed: Page Timeout

**Reply by mgineer85** [reply]
I am really sorry, but I cannot support Bluetooth devices and their coupling issues to raspberry system. I think you might experience more support in the raspberry forum for example.

**Reply by Striker62** [reply]
ok thanks . what Raspberry and os do you use

**Reply by Striker62** [reply]
only if it was not clear I'm talking about the photobooth-app-buzzer

**Reply by Striker62** [reply, marked as answer]
I solved it. I installed the software "blueman" and do the pairing via this.
Now it is connected
![bluetooth1](https://github.com/photobooth-app/photobooth-app/assets/37927465/5eef8e41-0d3b-4abe-b8d5-e8482ce6cb17)
![bluetooth2](https://github.com/photobooth-app/photobooth-app/assets/37927465/5cf6eead-2f44-465d-acd0-3617b5d6600b)
